### ISSUE TITLE
`WPS617` false positive with keyword args

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3596]
```python
self._media_by_precedence = sorted(
    (
        media_type
        for parser in self._parsers.values()
        if (media_type := MediaType(parser.content_type)).quality != 0
    ),
    key=lambda media: (media.specificity, media.quality),  # noqa: WPS617
    reverse=True,
)
```

It says:

```
django_modern_rest/negotiation.py

  52:17    WPS617 Found lambda assigned as an attribute
  key=lambda media: (media.specificity, media.quality),
      ^
```

Which is clearly a bug, it is a named param, not an attribute.
PRs are welcome, this is easy to fix.

**Comment by WYSIATI** [comment]
## 🔥 I'LL FIX THIS IN 24H FOR $100 (LOWEST PRICE)

**Undercutting everyone. Fastest delivery.**

✅ Fix in 24 hours (not 1 week)
✅ $100 (lowest offer)
✅ Pay AFTER it works
✅ Clean PR with tests

**Why wait? I'll start NOW.**

Reply YES → I code TODAY → You pay TOMORROW

No other proposal can beat this.
